tests/test-backend-ops.cpp

```diff
@@ -539,7 +539,7 @@ struct test_case {
         return false;
     }
 
-    bool eval_perf(ggml_backend_t backend, const char * op_name) {
+    bool eval_perf(ggml_backend_t backend1, ggml_backend_t backend2, const char * op_name) {
```

Comment:
I don't understand the purpose of this change, the perf mode only uses one backend.